deployment/docker-swarm/README.md

```diff
@@ -0,0 +1,513 @@
+# Black Duck Alert On Docker Swarm
+
+This document describes how to install and upgrade Alert in Docker Swarm.
+
+## Table Of Contents
+- [Requirements](#requirements)
+- [Installing Alert](#installing-alert)
+    - [Standalone Installation](#standalone-installation)
+    - [Installation With Black Duck](#installation-with-black-duck)
+- [Upgrading Alert](#upgrading-alert)
+    - [Standalone Upgrade](#standalone-upgrade)
+    - [Upgrade With Black Duck](#upgrade-with-black-duck)
+- [Certificates](#certificates)
+- [Environment Variables](#environment-variables) 
+    - [Edit Environment File](#edit-environment-file)
+    - [Edit the Overrides File](#edit-the-overrides-file)
+    - [Environment Variable Overrides](#environment-variable-overrides)
+    - [Alert Hostname Variable](#alert-hostname-variable)
+    - [Alert Logging Level Variable](#alert-logging-level-variable)
+    - [Email Channel Environment Variables](#email-channel-environment-variables)
+    - [Environment Variable Classifications](#environment-variable-classifications)
+- [Advanced Configuration](#advanced-configuration)
+    - [Changing Server Port](#changing-server-port)
+    - [Changing Memory Settings](#changing-memory-settings)
+
+## Requirements
+
+- You have a Docker host with at least 2GB of allocatable memory.
```

Comment:
This README seems to have some of the same issues Gavin mentioned earlier. Will probably want to make those same changes here as well.